record/record_test.go

```diff
@@ -48,6 +48,22 @@ var _ = Describe("Out New Relic", func() {
 			Expect(version).To(Equal(pluginVersion))
 			Expect(source).To(Equal("BARE-METAL"))
 		})
+
+		It("Doesn't rewrite plugin.type if it exits", func() {
+			inputMap := make(FluentBitRecord)
+			var inputTimestamp interface{}
+			inputTimestamp = output.FLBTime{
+				time.Now(),
+			}
+			expectedType := "something"
+			pluginMapIn := make(map[string]string)
+			pluginMapIn["type"] = expectedType
+			inputMap["plugin"] = pluginMapIn
```

Comment:
You can also write this in this compacted way:
```
inputMap["plugin"] := map[string]string{
    "type": expectedType,
}
```

I'd also suggest to rename `inputMap` into `inputRecord`, and `foundOutput` into `remappedRecord`.

Reply:
I just realize you used the same naming as in the other tests. Take this last suggestion as a pure recommendation, then. Feel free to also rename them in the rest of the tests.

Reply:
Done!